docs/UserGuide.md

```diff
@@ -136,28 +162,28 @@ An example:
 
 1) Create a workload with gold policy.
 
-```
-curl -H "Content-Type: application/json" --request POST --data \
+```shell
+$ curl -H "Content-Type: application/json" --request POST --data \
          '{"task_ids":["78377"], "policy": "gold"}' \
          http://127.0.0.1:8888/v1/workloads
 ```
 
 2) Create workload with max_cache, min_cache.
 
-```
-curl -H "Content-Type: application/json" --request POST --data \
+```shell
+$ curl -H "Content-Type: application/json" --request POST --data \
          '{"task_ids":["14988"], "max_cache": 4, "min_cache": 4}' \
          http://127.0.0.1:8888/v1/workloads
 ```
 
 Admin can change and add new policies by editing an toml/yaml file which is
 pointed in the configuration file.
 
-```
+```YAML
 policypath = "etc/rmd/policy.toml"
```

Comment:
I think this is bad design, the code require the config file any time, that is to say, if user don't install the config file, rmd will fail......

As far as I know, the issue existed for another config file too, like https://github.com/intel/rmd/blob/master/etc/rmd/cpu_map.toml

I would like to cleanup them before release rmd v1, this will allow user to quick run rmd after download the binary (default setting are contained in the binary)

No changes require for the guide, I will start set default setting in the code in later patch.

Thanks, this reminds me.